docs/tutorials/transforms.ipynb

```diff
@@ -111,12 +85,11 @@
     "import torchvision.transforms as T\n",
     "from PIL import Image\n",
     "from torch.utils.data import DataLoader\n",
+    "from copy import deepcopy\n",
```

Comment:
I guess isort doesn't currently support jupyter notebooks like black does. I wouldn't bother doing this manually until it does unless you really care. Trying not to be too anal about stuff like that.